### PR TITLE
Add tests for safelisting simple range headers from preflight

### PR DESCRIPTION
**Pull request by rayankans**
Fetch PR: https://github.com/whatwg/fetch/pull/1312/

### CODE REVIEW

File: cors/cors-safelisted-request-header.any.js [hunk 1]
```diff
@@ -40,4 +40,18 @@ function safelist(headers, expectPreflight = false) {
...
 ].forEach(([mimeType, preflight = false]) => {
   safelist({"content-type": mimeType}, preflight);
-})
+});
+
+[
+  ["100-200", true],
+  ["MB=100-200", true],
+  ["bytes=100-200"],
+  ["BYTES=100-200"],
```
Comment (jakearchibald) [inline cors/cors-safelisted-request-header.any.js:49]: The spec doesn't currently allow this. Is it case insensitive in the HTTP spec?

File: cors/cors-safelisted-request-header.any.js [hunk 2]
```diff
@@ -40,4 +40,18 @@ function safelist(headers, expectPreflight = false) {
   ["multipart/form-data;\"", true]
 ].forEach(([mimeType, preflight = false]) => {
   safelist({"content-type": mimeType}, preflight);
-})
+});
+
+[
+  ["100-200", true],
+  ["MB=100-200", true],
+  ["bytes=100-200"],
```
Comment (jakearchibald) [inline cors/cors-safelisted-request-header.any.js:48]:
```suggestion
  ["bytes=100-200"],
  ["bytes=100-200hello", true],
  ["bytes=abc-def", true],  
```

File: cors/cors-safelisted-request-header.any.js (hunk 1 above)
Comment (rayankans) [reply cors/cors-safelisted-request-header.any.js]: I thought headers were generally case-insensitive. I removed the test case.

**Review by annevk — COMMENTED** [review]
Can you also add one with `bytes `&nbsp;(followed by a space rather than `=`) as that's a somewhat common source of errors. Maybe also a leading `,` and whitespace around `=`.

**Comment by rayankans** [comment]
@annevk I added the test cases you suggested

**Review by annevk — APPROVED** [review]
Thanks!

**Pull request merged**